src/utils/url-helper.js

```diff
@@ -65,3 +68,21 @@ export function areEquivalentLocations(routeA, routeB) {
     query: routeBQuery,
   }));
 }
+
+// Resolve a given relative path into a full, absolute URL.
+//
+// @param {String} relativePath A relative path.
+// @return {String} The absolute URL corresponding to the given path.
+//
+//
+// Note that the same call may result in different output for the same input
+// depending on where/how this instance of DocC-Render is being hosted.
+//
+// Example:
+//
+// resolveAbsoluteUrl('/foo/bar') // http://localhost:8080/foo/bar
+// resolveAbsoluteUrl('/foo/bar') // https://mportiz08.github.io/example/foo/bar
+//
+export function resolveAbsoluteUrl(relativePath) {
```

Comment:
This wont work with a relative path though.

```
normaliseAssetUrl('/foo/bar') => '/my/base/foo/bar'
normaliseAssetUrl('foo/bar') => 'foo/bar'
normaliseAssetUrl('../foo/bar') => '../foo/bar'
```

So we need to make sure we use absolute **paths**.
```
new URL('foo/bar', 'http://website.com/path/one') => 'http://website.com/path/foo/bar' (notice how it removes "one", but keeps "path")
new URL('/foo/bar', 'http://website.com/path/one') => 'http://website.com/foo/bar' - what we want.

Reply:
a94761e3727ca8ea893c4e9de44a2500a5437530